contracts/modules/Offers/Omnibus/OffersOmnibus.sol

```diff
@@ -0,0 +1,325 @@
+
+    /// @notice Emitted when an offer is canceled
+    /// @param tokenContract The ERC-721 token address of the canceled offer
+    /// @param tokenId The ERC-721 token ID of the canceled offer
+    /// @param id The ID of the canceled offer
+    /// @param offer The metadata of the canceled offer
+    event OfferCanceled(address indexed tokenContract, uint256 indexed tokenId, uint256 indexed id, FullOffer offer);
+
+    /// @notice Emitted when an offer is filled
+    /// @param tokenContract The ERC-721 token address of the filled offer
+    /// @param tokenId The ERC-721 token ID of the filled offer
+    /// @param id The ID of the filled offer
+    /// @param taker The address of the taker who filled the offer
+    /// @param finder The address of the finder who referred the offer
+    /// @param offer The metadata of the filled offer
+    event OfferFilled(address indexed tokenContract, uint256 indexed tokenId, uint256 indexed id, address taker, address finder, FullOffer offer);
+
+    /// @param _erc20TransferHelper The ZORA ERC-20 Transfer Helper address
+    /// @param _erc721TransferHelper The ZORA ERC-721 Transfer Helper address
+    /// @param _royaltyEngine The Manifold Royalty Engine address
+    /// @param _protocolFeeSettings The ZORA Protocol Fee Settings address
+    /// @param _weth The WETH token address
+    constructor(
+        address _erc20TransferHelper,
+        address _erc721TransferHelper,
+        address _royaltyEngine,
+        address _protocolFeeSettings,
+        address _weth
+    )
+        IncomingTransferSupportV1(_erc20TransferHelper)
+        FeePayoutSupportV1(_royaltyEngine, _protocolFeeSettings, _weth, ERC721TransferHelper(_erc721TransferHelper).ZMM().registrar())
+        ModuleNamingSupportV1("Offers Omnibus")
+    {
+        erc721TransferHelper = ERC721TransferHelper(_erc721TransferHelper);
+    }
+
+    /// @notice Implements EIP-165 for standard interface detection
+    /// @dev `0x01ffc9a7` is the IERC165 interface id
+    /// @param _interfaceId The identifier of a given interface
+    /// @return If the given interface is supported
+    function supportsInterface(bytes4 _interfaceId) external pure returns (bool) {
+        return _interfaceId == type(IOffersOmnibus).interfaceId || _interfaceId == 0x01ffc9a7;
+    }
+
+    /// @notice Creates a simple WETH offer for a given NFT
+    /// @param _tokenContract The address of the ERC-721 token
+    /// @param _tokenId The id of the ERC-721 token
+    function createOfferMinimal(address _tokenContract, uint256 _tokenId) external payable nonReentrant returns (uint256) {
+        uint256 _offerAmount = msg.value;
+        weth.deposit{value: msg.value}();
+        weth.transferFrom(address(this), msg.sender, _offerAmount);
+
+        require(weth.allowance(msg.sender, address(erc20TransferHelper)) >= _offerAmount, "INSUFFICIENT_WETH_ALLOWANCE");
+        require(erc721TransferHelper.isModuleApproved(msg.sender), "MODULE_NOT_APPROVED");
```

Comment:
This shouldn't be necessary here, since the person making the offer is getting the transfer, I think.